README.md

```diff
@@ -265,6 +265,59 @@ Endpoints
         }, callback);
 
     ---
+**localizeServices.documents**
+
+    localizeService.documents.createDocument({
+        projectKey: <your projectkey>,
+    }, callback);
+    
+    Required fields:  projectKey 
+
+    ---
+    
+    localizeService.documents.createTranslation({
+        projectKey: <your projectkey>,
+    }, callback);
+    
+    Required fields:  projectKey 
+
+    ---
+ 
+    localizeService.documents.getDocuments({
+        projectKey: <your projectkey>,
+    }, callback);
+    
+    Required fields:  projectKey    
+    
+    ---
+   
+    localizeService.documents.downloadDocument({
+        projectKey: <your projectkey>,
+        documentId: <your documentId>,
+    }, callback);
+    
+    Required fields:  projectKey, documentId
+    
+    ---
+    
+    localizeService.documents.downloadTranslation({
+        projectKey: <your projectkey>,
+        documentId: <your documentId>,
+        language: 'fr',
```

Comment:
Should this follow the same patter and other variables and be `language: <your language>`

Reply:
I specifically did it this way to show the format of the code. The stuff  as `<your projectKey>` are referencing MongoIds.

Reply:
Got it, makes sense.